pkg/errors/errors.go

```diff
@@ -28,6 +28,7 @@ var (
 	InvalidVersion       error = errors.New("InvalidVersion")
 	OldVersion           error = errors.New("OldVersion")
 	OutOfCapacity        error = errors.New("OutOfCapacity")
+	NodeOutOfCapacity    error = errors.New("NodeOutOfCapacity")
```

Comment:
I think you forgot to push an update.  it should be:
OutOfNodeCapacity

Reply:
Opened a different PR: https://github.com/microsoft/moc/pull/145/files as this was saying that everything is upto date when I tried pushing